test/controllers/api/v2/hosts_controller_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'katello_test_helper'
+
+class Api::V2::HostsControllerTest < ActionController::TestCase
+  def setup
+    setup_foreman_routes
+    login_user(User.find(users(:admin)))
+    models
+    backend_stubs
+  end
+
+  def backend_stubs
+    Katello::Candlepin::Consumer.any_instance.stubs(:compliance_reasons).returns([])
+  end
+
+  def models
+    @content_view = katello_content_views(:acme_default)
+    @environment = katello_environments(:library)
+    @system = katello_systems(:simple_server)
+  end
+
+  def host_index_and_show(host)
```

Comment:
seemed simpler in this instance was a result of DRY'ing the test out.  Since its a 'simple' test just to make sure no errors occur seemed fine to me.